app/views/movies/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<% if @movie && @movie.errors.any? %>
+    <% @movie.errors.full_messages.each do |msg| %>
+
+      <%= msg %></br>
+<% end %>
+<% end %>
+
+
+<%= form_for(@movie, method: 'post') do |f| %>
+<%= f.label :title %>
+<%= f.text_field :title %></br></br>
+<%= f.label :description %>
+  <%= f.text_area :description, :class => "tinymce", :rows => 40, :cols => 120 %>
+  <%= tinymce %>
```

Comment:
There is already a text_area for tinymce field, Is it required here?

Reply:
yes sir, it is required to initialize tinymce, without this there is only a text area but no text editor.